skaffold.yaml.template

```diff
@@ -46,15 +46,15 @@ deploy:
     # datarepo-ui
     - name: TEMP-jade-datarepo-ui
       chartPath: https://github.com/broadinstitute/datarepo-helm/releases/download/datarepo-ui-0.0.12/datarepo-ui-0.0.12.tgz
-      version: 0.0.12
+      version: 0.0.15
```

Comment:
Given this and UI thing are gonna start getting bumped a lot now, should we just have the bumper upgrade these values?

Reply:
That's a great point! The template usually gets forgotten, so that would be great to add it to an action.